src/controllers/MessageController.js

```diff
@@ -698,14 +696,14 @@ export class MessageController extends EventEmitter {
           message.account
         );
 
-        const matcherFeeData = !hasFee
-          ? await this._getFee(message, result.data)
-          : {};
+        const matcherFee =
+          message.data.data?.matcherFee ||
```

Comment:
`?.` will it work here?

Reply:
Why not? It didn't work earlier because of old babel. Now it works